src/utils/api.js

```diff
@@ -47,14 +68,11 @@ const fetchWithHeaders = async (
       .concat(`${url.includes('?') ? '&' : '?'}`)
       .concat('_summary=count');
   }
-  const token = store.getState().app.token;
+  const credentials = setCredentials(fullUrl);
   return fetch(`${fullUrl}`, {
     signal: abortController ? abortController.signal : null,
-    headers: {
-      ...headers,
-      Authorization: `Basic ${token}`,
-      'Cache-Control': 'max-age=3600',
-    },
+    headers: getHeaders(headers),
+    credentials,
```

Comment:
I think `credentials` should be an object with key `credentials` and string value and not just a string